src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProfileSteps.java

```diff
@@ -94,14 +105,14 @@ public void verifyMyUserNameIsUpdatedInTheTopMenu() {
     public void verifyUserInformationIsUpdatedInUserDropdownMenu() {
         Map<String, String> dropdownMenuInfo = profilePage.getTopMenu().getUserMenu().getUserInformationAsMap();
         SoftAssert softAssert = new SoftAssert();
-        softAssert.assertEquals("Underlying Initials of User Menu does not match with the Initials edited previously.",
-                dropdownMenuInfo.get("Underlying initials"), user.getInitials());
-        softAssert.assertEquals("The Username from User Menu does not match with the Username edited previously.",
-                dropdownMenuInfo.get("Details user name"), user.getUserName());
-        softAssert.assertEquals("The Name from User Menu does not match with the Name edited previously.",
-                dropdownMenuInfo.get("Details name"), user.getName());
-        softAssert.assertEquals("The Initials from User Menu does not match with the Initials edited previously.",
-                dropdownMenuInfo.get("Details initials"), user.getInitials());
+        softAssert.assertEquals(dropdownMenuInfo.get("Underlying initials"), user.getInitials(),
```

Comment:
If you will validate all the inforamtion in the step just make sure the entity also has all these values